jsonschema-validation.xml

```diff
@@ -740,6 +740,23 @@
                 </t>
             </section>
 
+            <section title='"readOnly"'>
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the
+                    instance is managed exclusively by the server or the owning authority, and
+                    attempts by a user agent to modify the value of this property are expected to be
+                    ignored or rejected by a server.
```

Comment:
Let's change "by a server" to "by that owning authority" or something similar.  We want to emphasize the generality of the concept beyond client/server hypermedia.